forch/port_state_manager.py

```diff
@@ -188,6 +192,28 @@ def handle_static_device_behavior(self, mac, device_behavior):
             self._auto_sequester[mac_lower] = auto_sequester
             if device_behavior.segment:
                 self.handle_device_behavior(mac_lower, device_behavior, static=True)
+            sequester_scheduled = self._scheduled_sequester_timer.get(mac_lower)
+            if device_behavior.scheduled_sequestering_timestamp and not sequester_scheduled:
+                try:
```

Comment:
extract function here -- it would be nice to keep all the tz/date stuff isolated in one place, and keep the handle_static_device_bahavior function shorter.